waspc/src/Wasp/Util/IO.hs

```diff
@@ -74,3 +79,12 @@ deleteFileIfExists filePath = do
   let filePathStr = SP.toFilePath filePath
   exists <- SD.doesFileExist filePathStr
   when exists $ SD.removeFile filePathStr
+
+doesFileExist :: Path' r (File f) -> IO Bool
+doesFileExist = SD.doesFileExist . SP.toFilePath
+
+readFile :: Path' r (File f) -> IO String
+readFile = P.readFile . SP.toFilePath
+
+removeFile :: Path' r (File f) -> IO ()
+removeFile = SD.removeFile . SP.toFilePath
```

Comment:
Talked with @Martinsos, we're keeping these functions here for now. We thought about "mimicking" the structure of a package to add extra functionality (i.e., making a `System.Directory.Extra` module), but not all functionality defined here comes from the same package.

Anyway, let's just keep it here and extract it when the need arises.